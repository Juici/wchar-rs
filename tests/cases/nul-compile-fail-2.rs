use wchar::wch_c;

fn main() {
    wch_c!("also fails \0");
}
