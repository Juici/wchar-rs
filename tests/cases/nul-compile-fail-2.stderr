error: C-style string cannot contain nul characters
 --> $DIR/nul-compile-fail-2.rs:4:12
  |
4 |     wch_c!("also fails \0");
  |            ^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
